src/org/loklak/api/search/GithubProfileScraper.java

```diff
@@ -66,53 +70,42 @@ public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
 	public JSONObject serviceImpl(Query call, HttpServletResponse response, Authorization rights, JSONObjectWithDefault permissions)
 			throws APIException {
 		String profile = call.get("profile", "");
+		String termsS = call.get("terms", "");
```

Comment:
@sudheesh001 Actually just after this I have declared a Set named terms, so if I change termsS to terms, java is showing an exception : variable terms is aldready defined at  ```Set terms = null ```.